src/vanilla/Model/ResponseJv.cs

```diff
@@ -184,68 +179,37 @@ public virtual string GenericBodyClientTypeString
             }
         }
 
+        // The virtual properties below seem redundant in vanilla, but they are needed to allow fine-grained distinctions when generating paging methods in Azure.
+
+        /// <summary>
+        /// The generic type argument to a method overload with a ServiceFuture return type.
+        /// </summary>
         [JsonIgnore]
-        public virtual string ServiceFutureGenericParameterString
-        {
-            get
-            {
-                return GenericBodyClientTypeString;
-            }
-        }
+        public virtual string ServiceFutureGenericParameterString => GenericBodyClientTypeString;
 
+        /// <summary>
+        /// The generic type argument to a method overload with an Observable or Single return type.
+        /// </summary>
         [JsonIgnore]
-        public virtual string ServiceResponseGenericParameterString
-        {
-            get
-            {
-                return ServiceFutureGenericParameterString;
-            }
-        }
+        public virtual string ServiceResponseGenericParameterString => GenericBodyClientTypeString;
 
+        /// <summary>
+        /// The generic type argument to a service interface method, which uses concrete types for serialization.
+        /// </summary>
         [JsonIgnore]
-        public string XmlWrapperGenericParameterString
-        {
-            get
-            {
-                return Body.XmlName + "Wrapper";
-            }
-        }
+        public virtual string ServiceResponseConcreteParameterString => GenericBodyClientTypeString;
 
         [JsonIgnore]
-        public virtual string GenericHeaderClientTypeString
-        {
-            get
-            {
-                return HeaderClientType.ResponseVariant.Name;
-            }
-        }
+        public virtual string GenericHeaderClientTypeString => HeaderClientType.ResponseVariant.Name;
 
         [JsonIgnore]
-        public virtual string GenericBodyWireTypeString
-        {
-            get
-            {
-                return BodyWireType.Name;
-            }
-        }
+        public virtual string GenericBodyWireTypeString => BodyWireType.Name;
 
         [JsonIgnore]
-        public string GenericBodyWireTypeStringWrapped
-        {
-            get
-            {
-                return GenericBodyWireTypeString;
-            }
-        }
+        public string GenericBodyWireTypeStringWrapped => GenericBodyWireTypeString;

```

Comment:
What's the purpose of having "Wrapped" properties?

Reply:
This one isn't virtual so there's no conceivable purpose. Removing it.